xtokens/src/mock.rs

```diff
@@ -123,19 +123,29 @@ decl_test_parachain! {
 			pub type LocalOriginConverter = (
 				SovereignSignedViaLocation<LocationConverter, Origin>,
 				RelayChainAsNative<RelayChainOrigin, Origin>,
-				SiblingParachainAsNative<cumulus_pallet_xcm_handler::Origin, Origin>,
+				SiblingParachainAsNative<cumulus_pallet_xcm::Origin, Origin>,
 				SignedAccountId32AsNative<ParaANetwork, Origin>,
 			);
 
+			parameter_types! {
+				pub UnitWeightCost: Weight = 1_000;
+				pub const WeightPrice: (MultiLocation, u128) = (MultiLocation::X1(Junction::Parent), 1_000);
+				pub AllowUnpaidFrom: Vec<MultiLocation> = vec![ MultiLocation::X1(Junction::Parent) ];
+			}
+
 			pub struct XcmConfig;
 			impl XcmConfigT for XcmConfig {
 				type Call = Call;
-				type XcmSender = XcmHandler;
+				type XcmSender = ();
 				type AssetTransactor = LocalAssetTransactor;
 				type OriginConverter = LocalOriginConverter;
-				type IsReserve = MultiNativeAsset;
+				type IsReserve = ();
```

Comment:
The `XcmHandler` and `MultiNativeAsset` in config is essential for unit tests, please get them back.